File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int lengthNum = nums.size();
        int calculate = (lengthNum*(lengthNum+1))/2;
        int sum = 0;
        for(int i=0;i<nums.size();i++){
            sum  = sum + nums[i];
        }
        return calculate - sum;
    }
};

File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    const total = nums.length;
    const calculate = (total*(total+1))/2;
    let sum =0;
    for(let i=0;i<nums.length;i++){
        sum  = sum + nums[i];
    }  
    return calculate - sum;
};